tests/chainer_tests/test_optimizer.py

```diff
@@ -86,9 +86,9 @@ def setUp(self):
     def setup_cpu(self):
         self.optimizer.setup((self.params, self.grads))
 
-    def setup_gpu(self):
-        self.params = list(map(cuda.to_gpu, self.params))
-        self.grads = list(map(cuda.to_gpu, self.grads))
+    def setup_gpu(self, dst_id):
+        self.params = list(map(lambda p: cuda.to_gpu(p, dst_id), self.params))
+        self.grads = list(map(lambda p: cuda.to_gpu(p, dst_id), self.grads))
```

Comment:
I think that `self.grads = [cuda.to_gpu(p, dst_id) for p in self.grads]` is simple.